hazelcast/src/main/java/com/hazelcast/scheduledexecutor/ScheduledTaskStatistics.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.scheduledexecutor;
+
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.spi.annotation.Beta;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Statistics and timing info for a {@link IScheduledFuture} accessible through {@link IScheduledFuture#getStats()}
+ */
+@Beta
+public interface ScheduledTaskStatistics extends IdentifiedDataSerializable {
+
+    /**
+     * Returns how many times the task was ran/called.
+     *
+     * @return The numbers of runs
+     */
+    long getTotalRuns();
+
+    /**
+     * @return The monotonic creation time the task.
```

Comment:
What unit is this?
Probably good to refer to System.currentMillis 'monotonic creation time' is not revealing to me. Perhaps I'm too stupid :)